src/main/webapp/app/grading-system/grading-system.component.html

```diff
@@ -12,6 +12,17 @@ <h2 style="margin-bottom: 0.5em">
             </button>
             <jhi-grading-system-info-modal></jhi-grading-system-info-modal>
         </div>
+        <div class="d-flex">
+            <label class="btn btn-primary me-1 mb-1">
+                <span jhiTranslate="artemisApp.gradingSystem.csv.importButton">Import (.csv)</span>
+                <input type="file" accept=".json,.csv" style="display: none" (change)="onCSVFileSelect($event)" />
+            </label>
+        </div>
+        <div class="d-flex">
+            <button (click)="exportGradingStepsToCsv()" class="btn btn-primary me-1 mb-1" type="button">
+                <span jhiTranslate="artemisApp.gradingSystem.csv.exportButton">Export (.csv)</span>
+            </button>
+        </div>
```

Comment:
**Layout:** Maybe put all buttons in one line?

Reply:
![image](https://user-images.githubusercontent.com/73833780/141529275-d48e628e-fc50-4793-8662-b1565d98da12.png)